specification/cognitiveservices/data-plane/Language/stable/2022-05-01/conversationalanalysisauthoring.json

```diff
@@ -0,0 +1,3060 @@
+          }
+        }
+      },
+      "get": {
+        "description": "Gets the details of a project.",
+        "operationId": "ConversationalAnalysisAuthoring_GetProject",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The metadata of the project.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringProjectMetadata"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Get Project": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulGetProject.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Deletes a project.",
+        "operationId": "ConversationalAnalysisAuthoring_DeleteProject",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Successfully triggered the project deletion operation.",
+            "headers": {
+              "operation-location": {
+                "description": "The location of the status API for monitoring the created job.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Delete Project": {
+            "$ref": "./examples/conversationalanalysisauthoring/SuccessfulDeleteProject.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/authoring/analyze-conversations/projects/{projectName}/:export": {
+      "post": {
+        "description": "Triggers a job to export a project's data.",
+        "operationId": "ConversationalAnalysisAuthoring_Export",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "$ref": "#/parameters/ConversationalAnalysisAuthoringFormatInQuery"
+          },
+          {
+            "$ref": "#/parameters/ConversationalAnalysisAuthoringStringIndexTypeInQuery"
```

Comment:
We should already have a parameter for this in common.json. Or is that in-body? If it is in-body, you could add one for the query (though we use "Parameter" as a suffix for consistency).

Reply:
The authoring supports a subset of the StringIndexType values as mentioned in the other comment.